agent/bench-scripts/pbench-user-benchmark

```diff
@@ -69,15 +73,13 @@ while true; do
                 shift;
                 if [ -n "$1" ]; then
 			sysinfo="$1"
-			pbench-collect-sysinfo --sysinfo=$sysinfo --check
-			if [ $? == 0 ]; then
-				:
-			else
-				exit 1
-			fi
 			shift;
                 fi
                 ;;
+		-h|--help)
+		usage
+		exit 0;
```

Comment:
; is not necessary though it doesn't really matter. Everything else looks good to me.